elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -111,7 +127,55 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Let's have a comment like `<!-- Junit -->` here and for all other dependencies you added.